### ISSUE TITLE
Create Folder returns Bad Request

### ISSUE THREAD

**Issue by NicoTexas** [Jaspersoft/jrs-rest-java-client#294]
Trying to create a simple folder as superuser

```Java
        ClientFolder folder = new ClientFolder();
        String parentUri = "/public";
        folder.setUri("/public/test")
                .setLabel("test Folder")
                .setDescription("test folder description")
                .setPermissionMask(0)
                .setCreationDate("2017-01-24 16:27:47")
                .setUpdateDate("2017-01-24 16:27:47")
                .setVersion(0);

        OperationResult<ClientResource> result = getJasperSession()
                .resourcesService()
                .resource(folder.getUri())
                .createOrUpdate(folder);
```

Returns
com.jaspersoft.jasperserver.jaxrs.client.core.exceptions.BadRequestException: Bad Request

Also, is there a way to get more info than this in the response?

**Comment by TanyaEf** [comment]
You  can find log file in logs folder located in your application folder
(or change logging settings into console) where you find http logs with
error descriptor.

5 мая 2017 г. 23:23 пользователь "NicoTexas" <notifications@github.com>
написал:

**Comment by NicoTexas** [comment]
Ok, I found out how to turn on the http logs
This is the error I got
2 < 400
2 < Cache-Control: private
2 < Connection: close
2 < Content-Type: application/errorDescriptor+json
2 < Date: Thu, 11 May 2017 21:54:52 GMT
2 < Expires: Thu, 01 Jan 1970 00:00:00 UTC
2 < P3P: CP="ALL"
2 < Server: Apache-Coyote/1.1
2 < Transfer-Encoding: chunked
{"message":"The value \"2014-01-24 16:27:47\" for parameter \"creationDate\" is invalid.","errorCode":"illegal.parameter.value.error","parameters":["creationDate","2017-01-24 16:27:47"]}

I tried to change that with the date straight from the doc 2014-01-24 16:27:47 and got same error.

**Comment by TanyaEf** [comment]
try not to set this parameter at all, it's set automatically by server(as well as update date and version)

**Comment by NicoTexas** [comment]
Indeed, it works when not set but documentation example date format is wrong.
Thanks
